Standards/scs-0211-v1-kaas-default-storage-class.md

```diff
@@ -24,7 +24,7 @@ While often times, consumers will choose a `StorageClass` explicitly, usually, t
 
 This document attempts to define the properties this default `StorageClass` should have.
 
-## Decision
+## Standard
```

Comment:
I'm not happy with either variant here. This is turning out harder than I expected.

Reply:
See https://github.com/SovereignCloudStack/standards/pull/441#discussion_r1549764594